kproject-common/src/main/kotlin/com/soywiz/kproject/model/KProjectModel.kt

```diff
@@ -60,7 +60,7 @@ class KSource(
     fun resolveDir(): File {
         return when (parts.first()) {
             "git" -> {
-                val (_, repo, path, version) = parts
+                val (_, _, repo, path, version) = parts
```

Comment:
What does this do?

Reply:
I can  read from the readme that there are two src-line formats:

    git :: repo-path-on-github :: path-to-src :: version-or-commit-hash

    git :: name :: repo-path-on-github :: path-to-src :: version-or-commit-hash

One is with name and one it without 

It looks like that a kproject.yml file needs the name in the src line but a specific file like kds.kproject.yml does not need it.
This is a bit confusing because it is not consistent. Can we make it consistent?

I guess my change not needed just in case we want to make it consistent...